Q/qwtw/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "qwtw"
+version = v"2.0.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ig-or/qwtw.git", "b8119478c38050cd55e01711c8b5867a1932c990")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add g++
+cd $WORKSPACE/srcdir
+cd qwtw
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ../.
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+
+platforms = [
+    Platform("x86_64", "windows"),
+    Platform("x86_64", "linux"; libc="glibc"),
+
+]
+
+
```

Comment:
The log shows:
```
┌ Warning: /agent/_work/1/s/Q/qwtw/build/x86_64-w64-mingw32/QkPHcoCf/destdir/bin/libqwtw.dll contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/57NRU/src/auditor/compiler_abi.jl:242
```
```suggestion
]
platforms = expand_cxxstring_abis(platforms)
```
Apart from this, it looks good :+1:

Reply:
probably will not work with GCC 4.. looks like there are some C++14.
can we leave it as it is?

Reply:
What?

Reply:
Oh, the warning message says to build with GCC 4, uh, that's misleading.  Everything is built with GCC 8 here